admin/src/Components/Layout/Layout.tsx

```diff
@@ -0,0 +1,152 @@
+import * as React from "react";
+import { AppBar, ListItemButton, Button } from "@mui/material";
+import Box from "@mui/material/Box";
+import CssBaseline from "@mui/material/CssBaseline";
+import Divider from "@mui/material/Divider";
+import Drawer from "@mui/material/Drawer";
+import IconButton from "@mui/material/IconButton";
+import InboxIcon from "@mui/icons-material/MoveToInbox";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import ListItemText from "@mui/material/ListItemText";
+import MailIcon from "@mui/icons-material/Mail";
+import MenuIcon from "@mui/icons-material/Menu";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import { Routes, useNavigate, Route } from "react-router-dom";
+
+const drawerWidth = 240;
+
+interface Props {
+  /**
+   * Injected by the documentation to work in an iframe.
+   * You won't need it on your project.
+   */
+  window?: () => Window;
+}
+
+export default function LayoutB(props: Props) {
+  const { window } = props;
+  const [mobileOpen, setMobileOpen] = React.useState(false);
+
+  const handleDrawerToggle = () => {
+    setMobileOpen(!mobileOpen);
+  };
+
+  const navigate = useNavigate();
+  const drawer = (
+    <div>
+      <Toolbar />
+
+      <List>
+        <ListItem button onClick={() => navigate("/")}>
+          <ListItemText primary="Orders" />
+        </ListItem>
+
+        <ListItem button onClick={() => navigate("/products")}>
+          <ListItemText primary="Products" />
+        </ListItem>
+
+        <ListItem button onClick={() => navigate("/references")}>
+          <ListItemText primary="References" />
+        </ListItem>
+
+        <ListItem button onClick={() => navigate("/pages")}>
+          <ListItemText primary="Pages" />
+        </ListItem>
+
+        <ListItem button onClick={() => navigate("/users")}>
+          <ListItemText primary="Users" />
+        </ListItem>
+      </List>
+    </div>
+  );
+
+  const container =
+    window !== undefined ? () => window().document.body : undefined;
+
+  return (
+    <Box sx={{ display: "flex" }}>
+      <CssBaseline />
+      <AppBar
+        position="fixed"
+        sx={{
+          width: { sm: `calc(100% - ${drawerWidth}px)` },
+          ml: { sm: `${drawerWidth}px` },
+        }}
+      >
+        <Toolbar>
+          <IconButton
+            color="inherit"
+            aria-label="open drawer"
+            edge="start"
+            onClick={handleDrawerToggle}
+            sx={{ mr: 2, display: { sm: "none" } }}
+          >
+            <MenuIcon />
+          </IconButton>
+          <Typography variant="h6" noWrap component="div">
+            Responsive drawer
```

Comment:
This Text should change depending on which route we are currently at. Should display Orders when on /orders, Products when on /products etc.

Reply:
Moving this into a seperate issue so we can merge the branch right now.